templates/00_struct.go.tpl

```diff
@@ -19,7 +19,7 @@ type {{$alias.UpSingular}} struct {
 	{{end -}}
 }
 
-var {{$alias.UpSingular}}Columns = struct {
+var {{$alias.UpSingular}}AllColumns = struct {
```

Comment:
Spoke a bit too soon hah! This can't be changed, this is breaking and will affect all sqlboiler users that are currently referring to columns. Because this is an uppercase variant, I don't believe this needed to be changed in the first place. `UpdateColumns` and `updateColumns` cannot conflict. Will need this fixed before I can merge.